app/models/setting.rb

```diff
@@ -83,4 +83,23 @@ def self.invoice_reminder_in_days
   def self.wishlist_size
     Setting.find_by(code: :wishlist_size).value.to_i
   end
+
+  def self.violations_count_regulations_link
+    value = Setting.find_by(code: :violations_count_regulations_link)&.value
+    if json_hash?(value)
+      JSON.parse(value)[I18n.locale.to_s]
+    else
+      value.to_s
+    end
+  end
+
+  def self.json_hash?(text)
+    JSON.parse(text).is_a?(Hash)
+  rescue JSON::ParserError
+    false
+  rescue NoMethodError
```

Comment:
Such things usually lead to debugging nightmare, since you're effectively suppressing the errors. How about revamping the logic so that it doesn't require `rescue`ing? Perhaps it can be substituted by guard clause?